server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -167,8 +165,8 @@ public void testExpandsConfigurationWhenGrowingFromOneToThreeNodesAndShrinksOnFa
         {
             final ClusterNode newLeader = cluster.getAnyLeader();
             final VotingConfiguration lastCommittedConfiguration = newLeader.getLastAppliedClusterState().getLastCommittedConfiguration();
-            assertThat(lastCommittedConfiguration + " should be 1 node", lastCommittedConfiguration.getNodeIds().size(), equalTo(1));
-            assertFalse(lastCommittedConfiguration.getNodeIds().contains(disconnect1.getId()));
+            assertThat(lastCommittedConfiguration + " should be all nodes", lastCommittedConfiguration.getNodeIds(),
+                equalTo(cluster.clusterNodes.stream().map(ClusterNode::getId).collect(Collectors.toSet())));
```

Comment:
test name is not correct anymore? it says "AndShrinksOnFailure"

Reply:
The name of the test that contains this line is `testExpandsConfigurationWhenGrowingFromOneNodeToThreeButDoesNotShrink` which seems right to me?

Reply:
Github UI fooled me